src/TLuaInterpreter.cpp

```diff
@@ -13776,19 +13776,19 @@ void TLuaInterpreter::logError(std::string& e, const QString& name, const QStrin
 {
     // Log error to Editor's Errors TConsole:
     if (mpHost->mpEditorDialog) {
-        mpHost->mpEditorDialog->mpErrorConsole->print(QStringLiteral("[ERROR:]"), QColor(Qt::blue), QColor(Qt::black));
-        mpHost->mpEditorDialog->mpErrorConsole->print(QStringLiteral(" object:<%1> function:<%2>\n").arg(name, function), QColor(Qt::green), QColor(Qt::black));
+        mpHost->mpEditorDialog->mpErrorConsole->print(QStringLiteral("[%1:]").arg(tr("ERROR")), QColor(Qt::blue), QColor(Qt::black));
+        mpHost->mpEditorDialog->mpErrorConsole->print(QStringLiteral(" %1:<%2> %3:<%4>\n").arg(tr("object"), name, tr("function"), function), QColor(Qt::green), QColor(Qt::black));
```

Comment:
I "object" :) These single-word translations could be very hard to grasp in different languages. 
How about adding a descriptionary text as secondary argument to tr() in order to indicate exactly what "object" and "function" may refer to, are they verb or noun, etc. Maybe include the resulting line for reference. Also keep the same comment in their other line of code below, so the translations are not doubled in Crowdin.

Reply:
Yeah, one should avoid embedding individual translatable words into a larger fixed string - though there are *sometimes* justifiable grounds for doing so and I think in this case we may have hit one: the trouble is that we are colouring separate sections - I suspect we will have to have a rethink of the format for this type of message - and perhaps provide a different method to insert colouration into a translatable string...

Reply:
Correct, I don't want translators to be in control of the text formatting - they should only translate.

I'll add the context information!

Reply:
Added now.